### ISSUE TITLE
[Feature Request] Bump "vscode-json-languageservice" to 5.3.1

### ISSUE THREAD

**Issue by fhareide** [microsoft/monaco-editor#3625]
### Context

- [X] This issue is not a bug report. *(please use a different template for reporting a bug)*
- [X] This issue is not a duplicate of an existing issue. *(please use the [search](https://github.com/microsoft/monaco-editor/issues) to find existing issues)*

### Description

Would it be possible to bump the version of the json languageservice to 5.3.1? Right now it has an old version that does not validate any of the "newer" json schemas. With this new version (I have tested locally) it validates fine .

**Comment by jrsquared** [comment]
Any chance of getting this updated?

**Comment by fhareide** [comment]
Yeah I'm still waiting too. In the meantime I patched a version I can use while waiting. No guarantees, only needed it for a small project https://www.npmjs.com/package/@fhareide/monaco-editor
